src/main/java/uk/gov/hmcts/cmc/claimstore/courtfinder/CourtFinderApi.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.cmc.claimstore.courtfinder;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import uk.gov.hmcts.cmc.claimstore.courtfinder.models.Court;
+import uk.gov.hmcts.cmc.claimstore.courtfinder.models.CourtDetails;
+
+import java.util.List;
+
+@FeignClient(name = "court-finder-api", url = "https://courttribunalfinder.service.gov.uk")
```

Comment:
We might want to pass this as an env variable - probably this url is the same on aat/prod, but it might be worth being able to stub this out if we want to add wiremock tests and so on